features/aanschrijfwijze.feature

```diff
@@ -136,135 +141,622 @@ Functionaliteit: Als gemeente wil ik de juiste en consistente aanschrijfwijze va
+        | predikaat, voorvoegsel, partner geen voorvoegsel                           | N           | PK VL VV GN-GP    | 1         |
+        | predikaat, voorvoegsel, partner voorvoegsel                                | N           | PK VL VV GN-VP GP | 1         |
+        | adellijke titel, geen voorvoegsel, partner geen voorvoegsel                | N           | VL AT GN-GP       |           |
+        | adellijke titel, geen voorvoegsel, partner voorvoegsel                     | N           | VL AT GN-VP GP    |           |
+        | adellijke titel, voorvoegsel, partner geen voorvoegsel                     | N           | VL AT VV GN-GP    |           |
+        | adellijke titel, voorvoegsel, partner voorvoegsel                          | N           | VL AT VV GN-VP GP |           |
+        | geen adellijke titel/predikaat, geen voorvoegsel, partner geen voorvoegsel | V           | VL GP-GN          |           |
+        | geen adellijke titel/predikaat, geen voorvoegsel, partner voorvoegsel      | V           | VL VP GP-GN       |           |
+        | geen adellijke titel/predikaat, voorvoegsel, partner geen voorvoegsel      | V           | VL GP-VV GN       |           |
+        | geen adellijke titel/predikaat, voorvoegsel, partner voorvoegsel           | V           | VL VP GP-VV GN    |           |
+        | predikaat, geen voorvoegsel, partner geen voorvoegsel                      | V           | VL GP-PK GN       | 3         |
+        | predikaat, geen voorvoegsel, partner voorvoegsel                           | V           | VL VP GP-PL GN    | 3         |
+        | predikaat, voorvoegsel, partner geen voorvoegsel                           | V           | VL GP-PK VV GN    | 3         |
+        | predikaat, voorvoegsel, partner voorvoegsel                                | V           | VL VP GP-PK VP GP | 3         |
+        | adellijke titel, geen voorvoegsel, partner geen voorvoegsel                | V           | VL AT GN-GP       |           |
+        | adellijke titel, geen voorvoegsel, partner voorvoegsel                     | V           | VL AT GN-VP GP    |           |
+        | adellijke titel, voorvoegsel, partner geen voorvoegsel                     | V           | VL AT VV GN-GP    |           |
+        | adellijke titel, voorvoegsel, partner voorvoegsel                          | V           | VL AT VV GN-VP GP |           |
+
+        1. predikaat begint met hoofdletter en wordt geplaatst vóór de voorletters
+        2. predikaat wordt niet opgenomen bij naamgebruik P
+        3. predikaat begint met kleine letter en wordt geplaatst vóór eigen achternaam (voorvoegsel en geslachtsnaam) bij naamgebruik V
+        4. adellijke titel wordt geplaatst tussen voorletters en achternaam (voorvoegsel en geslachtsnaam)
+        5. adellijke titel wordt niet opgenomen bij naamgebruik P
+
```

Comment:
deze tabel bevat impliciet of als "opmerking" verschillende regels. Ik denk dat het goed is die regels expliciet op te nemen bij de rule. Ook ontbreken enkele regels:

bij aanduidingNaamgebruik='E' wordt samenstelling: PK VL AT VV GN
bij aanduidingNaamgebruik='P' wordt samenstelling: VL VP GP
bij aanduidingNaamgebruik='N' wordt samenstelling: PK VL AT VV GN-VP GP
bij aanduidingNaamgebruik='V' wordt samenstelling: VL VP GP-PKAT VV GN
wanneer een naamcomponent geen of een lege waarde heeft, wordt de overbodige spatie niet opgenomen: niet starten met een spatie, niet eindigen met een spatie, geen dubbele spatie (ook voorbeeld met alleen GN en lege VL en VV toevoegen)
voorvoegsel wordt in kleine letters geschreven, zie https://onzetaal.nl/taaladvies/hoofdletters-in-aanhef-brief-of-mail (staat niet in regel, voorbeelden zijn onjuist - trouwens ook in de oude versie feature)
adellijke titel of predikaat begint met een kleine letter (staat niet in de regel, blijkt alleen uit voorbeelden)
een adellijke titel of predikaat wordt opgenomen in de vorm die hoort bij de geslachtsaanduiding, wanneer die ontbreekt wordt de adellijke titel of predikaat niet opgenomen (staat niet in regel, wel in algemene toelichting v.a. regel 43, voorbeelden ontbreken)
de eerste letter van de eerste naamcomponent wordt met een hoofdletter geschreven (meestal VL of PK): bijv. bij PK, VL en AT hebben geen waarde, begint voorvoegsel met een hoofdletter (staat niet in regel, voorbeeld ontbreekt - trouwens ook in de oude versie feature)

Reply:
11. Wanneer de persoon een predikaat heeft en de geslachtsaanduiding is gelijk aan "V" (vrouw) en betrokkene heeft een partner, wordt het predikaat niet opgenomen
12. Wanneer de persoon een predikaat heeft en de geslachtsaanduiding is gelijk aan "V" (vrouw) en betrokkene gebruikt de naam van de (ex)partner (aanduidingNaamgebruik is ongelijk aan "E"), wordt het predikaat niet opgenomen